lib/css/CssParser.js

```diff
@@ -369,6 +371,59 @@ class CssParser extends Parser {
 			return pos;
 		};
 		const eatPropertyName = eatUntil(":{};");
+		const parseModulesValues = (input, pos) => {
+			let start = pos;
+			pos = walkCssTokens.eatWhitespaceAndComments(input, pos);
+			if (pos === input.length) return pos;
+			let key;
+			[pos, key] = eatText(input, pos, eatUntil(":{};/"));
+			if (pos === input.length) return pos;
+			if (input.charCodeAt(pos) !== CC_COLON) {
+				this._emitWarning(
+					state,
+					`Unexpected '${input[pos]}' at ${pos} during parsing of key in '@value' (expected ':')`,
+					locConverter,
+					start,
+					pos
+				);
+				return pos;
+			}
+			pos++;
+			if (pos === input.length) return pos;
+			pos = walkCssTokens.eatWhitespaceAndComments(input, pos);
+			if (pos === input.length) return pos;
+			let value;
+			[pos, value] = eatText(input, pos, eatUntil(":{};/"));
```

Comment:
I think it should be only `;` here? Do we really need parse it? Maybe we can parse eveyrything to `;`/`from` (future improvement)?

Reply:
if only eat until `;`, then how to solve the situation where user wirtes unexpected cahracter or the user forgets to write ';'

Reply:
Do you mean eat until ";" and collect the string between @value and ";" and then use a regex to match to `key` and `value` like `postcss-modules-values`, this would be more flexible, but maybe a little inappropriate